src/components/PageSections/Menu/_FullSiteMenu.scss

```diff
@@ -39,33 +37,46 @@
 
 .coa-ThemesNav__theme {
   margin: 0;
+  display: table;
+  position: relative;
+  text-align: left;
+  height: 56px;
+  padding: 10px 5px 10px 5px;
 }
 
 .coa-ThemesNav__link {
-  &:hover {
-    color: $coa-color-white;
-  }
+  transition: background-color .3s, box-shadow .3s;
+  display: table-cell;
+  vertical-align: middle;
+  margin: 10px;
+  border-radius: 18px;
+  padding-left: 16px;
+  padding-right: 16px;
   cursor: pointer;
   text-decoration: none;
   font-size: $coa-font-size-xsmall;
-  color: $coa-color-white;
+  font-weight: $coa-font-regular !important; // needs to override font-weight for a tags.
+  color: $coa-color-burnt-coffee-black;
   font-weight: $coa-font-medium;
```

Comment:
Would removing the font-weight on line 60 preclude the need for an !important font-weight on line 58?

Reply:
It absolutely would!